node_modules/oae-authentication/tests/test-auth-local.js

```diff
@@ -1173,55 +1205,75 @@ describe('Authentication', function() {
+                                    assert.ok(err);
+                                    assert.equal(err.code, 400);
+                                    assert.ok(!user);
 
-                                // Verify the context was not authenticated
-                                RestAPI.User.getMe(testGlobalUserRestContext, function(err, me) {
-                                    assert.ok(!err);
-                                    assert.ok(me.anon);
+                                    // Ensure the global admin still cannot be authenticated
+                                    var testGlobalUserRestContext = TestsUtil.createGlobalRestContext();
 
-                                    // Sanity check the user can be created through the REST endpoints
-                                    RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, null, function(err, user) {
-                                        assert.ok(!err);
-                                        assert.ok(user);
+                                    // Ensure that the credentials do not authenticate a value global administrator
+                                    RestAPI.Authentication.login(testGlobalUserRestContext, username, username, function(err) {
+                                        assert.ok(err);
+                                        assert.equal(err.code, 401);
 
-                                        // Ensure when we try and create one with the same loginId, we get a 400 error
-                                        RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, null, function(err, secondUser) {
-                                            assert.ok(err);
-                                            assert.equal(err.code, 400);
+                                        // Verify the context was not authenticated
+                                        RestAPI.User.getMe(testGlobalUserRestContext, function(err, me) {
+                                            assert.ok(!err);
+                                            assert.ok(me.anon);
 
-                                            return callback();
+                                            // Sanity check the user can be created through the REST endpoints
+                                            RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, email, {}, function(err, user) {
+                                                assert.ok(!err);
+                                                assert.ok(user);
+
+                                                // Ensure when we try and create one with the same loginId, we get a 400 error
+                                                RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, email, {}, function(err, secondUser) {
```

Comment:
I think it might so long as it doesn't force us to break some kind of privacy boundary. For example I type in an email address, now I get a list of accounts (private, loggedin, public) that are set up with that email address.

You also have to take into consideration private tenants. Not in the case that you're sharing something from a private tenant (I'm assuming that would not be allowed), but instead you've entered an email address that exists in the system but it's for a user who exists in a private tenant elsewhere. I guess you would just treat it as though there is no user with that email address in the system?